.github/workflows/main.yml

```
@@ -27,6 +28,31 @@ jobs:
         with:
           python-version: "3.10"
 
+      - name: Install XQuartz
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:24: Double quote to prevent globbing and word splitting

(shellcheck)